### ISSUE TITLE
Initial learning rate

### ISSUE THREAD

**Issue by liuxu-about** [Francis0625/Omni-SR#11]
The learning rate in train_OmniSR_X2_DF2K.yaml is 5e-4.
But in your Omni paper, the learning rate is set to 1e-4.
Please tell me the reason.

**Comment by Francis0625** [comment]
Sorry for the misalignment, and we will update the paper and the code. Using the provied training setting, you can achieve better results than the reported results in the paper.
